src/Microsoft.CodeAnalysis.Analyzers/Core/FixAnalyzers/FixerWithFixAllAnalyzer.cs

```diff
@@ -287,7 +287,7 @@ void AnalyzeFixerWithFixAll(INamedTypeSymbol fixer, SymbolAnalysisContext contex
                         {
                             if (IsViolatingCodeActionObjectCreation(objectCreation))
                             {
-                                Diagnostic diagnostic = objectCreation.CreateDiagnostic(OverrideCodeActionEquivalenceKeyRule, objectCreation.Constructor.ContainingType, EquivalenceKeyPropertyName);
+                                Diagnostic diagnostic = objectCreation.CreateDiagnostic(OverrideCodeActionEquivalenceKeyRule, objectCreation.Constructor!.ContainingType, EquivalenceKeyPropertyName);
```

Comment:
Is there a check that eliminates the possibility of https://github.com/dotnet/roslyn-analyzers/issues/6089?

Reply:
Oh it's below.